var/spack/repos/builtin/packages/py-celery/package.py

```diff
@@ -46,19 +47,26 @@ class PyCelery(PythonPackage):
     # 'cosmosdbsql',
     # 's3',
 
+    depends_on('python@3.7:', type=('build', 'run'), when="@5.2.3:")
     depends_on('python@2.7:2.8,3.4:', type=('build', 'run'))
 
     depends_on('py-setuptools', type=('build', 'run'))
 
     depends_on('py-redis@3.2.0:', when='+redis', type=('build', 'run'))
     depends_on('py-sqlalchemy', when='+sqlalchemy', type=('build', 'run'))
 
-    depends_on('py-click@7.0:7', when="@5.0.0:5", type=('build', 'run'))
+    depends_on('py-click@7.0:7', when="@5.0.0:5.0", type=('build', 'run'))
+    depends_on('py-click@8.0.3:8', when="@5.2.0:5.2", type=('build', 'run'))
     depends_on('py-click-didyoumean@:0.0.3', when="@5.0.0:5", type=('build', 'run'))
-    depends_on('py-click-repl@:0.1.6', when="@5.0.0:5", type=('build', 'run'))
+    depends_on('py-click-repl@:0.1.6', when="@5.0.0:5.0", type=('build', 'run'))
+    depends_on('py-click-repl@0.2.0:0.2', when="@5.2.0:5.2", type=('build', 'run'))
     depends_on('py-pytz@2019.3:', type=('build', 'run'))
+    depends_on('py-pytz@2021.3:', type=('build', 'run'), when="@5.2.3")
     depends_on('py-billiard@3.6.3:3', type=('build', 'run'))
+    depends_on('py-billiard@3.6.4:3', type=('build', 'run'), when="@5.2.3")
     depends_on('py-kombu@4.6.11', when="@4.3.0:4",  type=('build', 'run'))
-    depends_on('py-kombu@5.0.0:', when="@5.0.0:5", type=('build', 'run'))
+    depends_on('py-kombu@5.0.0:', when="@5.0.0:5.0", type=('build', 'run'))
+    depends_on('py-kombu@5.2.3:', when="@5.2.0:5.2", type=('build', 'run'))
     depends_on('py-vine@1.3.0', when="@4.3.0:4", type=('build', 'run'))
     depends_on('py-vine@5.0.0:', when="@5.0.0:5", type=('build', 'run'))
+    depends_on('py-importlib-metadata@0.18:', type=('build', 'run'), when='python@3.5:3.7')
```

Comment:
Where did you find this dep?

Reply:
I believe this was a mistake that belongs in kombu.